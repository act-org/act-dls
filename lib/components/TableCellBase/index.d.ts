/**
 * @prettier
 */
import * as React from 'react';
import { TableCellProps } from '@material-ui/core';
export declare type Props = TableCellProps;
declare const TableCellBase: React.FC<Props>;
export default TableCellBase;
//# sourceMappingURL=index.d.ts.map