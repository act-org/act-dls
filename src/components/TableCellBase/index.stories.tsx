/**
 * @prettier
 */

import * as React from 'react';
import { text } from '@storybook/addon-knobs';

import TableCellBase from '.';

export const base = (): React.ReactElement<any> => (
  <TableCellBase>{text('Text', 'Table Cell')}</TableCellBase>
);

export default {
  component: TableCellBase,
  parameters: {
    info: {
      text: 'This is the base variant of the TableCell component.',
    },
  },
  title: 'Atoms/TableCell',
};
