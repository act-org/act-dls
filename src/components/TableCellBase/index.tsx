/**
 * @prettier
 */

import * as React from 'react';
import { TableCell, TableCellProps } from '@material-ui/core';

export type Props = TableCellProps;

const TableCellBase: React.FC<Props> = (
  props: Props,
): React.ReactElement<any> => <TableCell {...props} />;

export default TableCellBase;
