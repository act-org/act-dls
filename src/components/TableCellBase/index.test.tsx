/**
 * @prettier
 */

import * as React from 'react';

import render from '~/helpers/test/render';

import TableCellBase from '.';

describe('TableCellBase', () => {
  it('matches the snapshot', () => {
    const { container } = render(
      <TableCellBase align="center">Table Cell</TableCellBase>,
    );

    expect(container).toMatchSnapshot();
  });
});
