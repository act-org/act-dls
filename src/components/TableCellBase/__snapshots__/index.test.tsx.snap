// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableCellBase matches the snapshot 1`] = `
<div>
  <td
    class="MuiTableCell-root MuiTableCell-alignCenter"
  >
    Table Cell
  </td>
</div>
`;
